.github/workflows/python-package.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install wheel "coverage>=4.0,<4.4"
+        pip install -r requirements.txt
+    - name: Test with coverage
+      run: |
+        coverage run --source=binaryiotools -m unittest discover -b 
+    - if: matrix.python-version == '3.7'
+      name: Publish coverage
+      uses: codecov/codecov-action@v1
+      with:
+        token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Can you revert this for the time being please?

You can probably git stash or copy and paste files if you want this downstream. Though it would be appreciated if you can git reset hard just to keep the commit history reasonably tidy